lib/tasks/assets.rake

```diff
@@ -6,7 +6,7 @@
 namespace :assets do
   desc "Create .gz versions of assets"
   task gzip: :environment do
-    Dir["#{Rails.root}/public/assets/**/*.{css,html,js,otf,svg,txt,xml}"].each do |file|
+    Dir[Rails.root.join("public", "assets", "**", "*.{css,html,js,otf,svg,txt,xml")].each do |file|
```

Comment:
@RamilyaNigmatullina Should we use `"*.{css,html,js,otf,svg,txt,xml}"` instead of `"*.{css,html,js,otf,svg,txt,xml"`?